tests/test_http_ext_auth.py

```diff
@@ -80,6 +80,69 @@
     "code_challenge_methods_supported": ["plain", "S256"],
 }
 
+AZURE_DISCOVERY_DOCUMENT = {
+    "token_endpoint": (
+        "https://login.microsoftonline.com/common/oauth2/v2.0/token"
+    ),
+    "token_endpoint_auth_methods_supported": [
+        "client_secret_post",
+        "private_key_jwt",
+        "client_secret_basic",
+    ],
+    "jwks_uri": "https://login.microsoftonline.com/common/discovery/v2.0/keys",
+    "response_modes_supported": ["query", "fragment", "form_post"],
+    "subject_types_supported": ["pairwise"],
+    "id_token_signing_alg_values_supported": ["RS256"],
+    "response_types_supported": [
+        "code",
+        "id_token",
+        "code id_token",
+        "id_token token",
+    ],
+    "scopes_supported": ["openid", "profile", "email", "offline_access"],
+    "issuer": "https://login.microsoftonline.com/{tenantid}/v2.0",
```

Comment:
Notice the weird template string here. This is a copy-paste from the _actual_ JSON response here, so _hopefully_ we never need to use this `issuer` property and just just safely ignore it. I feel like they should be hydrating this themselves since it was requested from a specific tenant (`common`) and the rest of the URLs present here are hydrated already.